ortho_seq_code/cli.py

```diff
@@ -1,13 +1,19 @@
 import click
-from ortho_seq_code.orthogonal_polynomial import cli as orthogonal_polynomial
-
+from ortho_seq_code.orthogonal_polynomial import ortho_poly_command
+from ortho_seq_code.gui.gui import gui_run
```

Comment:
One more blank line after this so it's two before the definition of the constant variable below.